docs/core/tutorials/using-mstest-on-windows.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Use MSTest with .NET Core on Windows
+description: How to use MSTest with .NET Core on Windows, using Visual Studio 2015
+keywords: MSTest, .NET, .NET Core
+author: 
```

Comment:
Can you put your GitHub ID here for author? Then, the build process will pick it up and add your attribution on the published site.